crates/giskard-server/static/app.css

```
@@ -109,6 +109,9 @@
   .thread.activity-running .thread-status { color:#d29922; }
   .thread.activity-approval .thread-status { color:#f0d98c; }
   .thread.activity-error .thread-status { color:#f85149; }
+  /* Hoisted from a hidden sub-agent: same urgency colour, outlined so the row still reads as
+     "something under here" rather than "this thread". */
+  .thread.activity-subagent .thread-status { text-shadow:0 0 6px currentColor; opacity:.9; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)